pkg/types/installconfig.go

```diff
@@ -139,6 +140,10 @@ type InstallConfig struct {
 	// GCP: "Mint", "Passthrough", "Manual"
 	// +optional
 	CredentialsMode CredentialsMode `json:"credentialsMode,omitempty"`
+
+	// FeatureGateSelection allows to enable additional unsupported features.
```

Comment:
I am not keen on putting configuration for unsupported features into the InstallConfig. Is there a compelling reason for why users that want to use feature gates cannot create the manifests themselves?